### PR TITLE
CD-RN-1.28.x

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
CD RN 1.28.x 
Delegate version 82500

### CODE REVIEW

**Review by krishi0408 — DISMISSED** [review]

File: release-notes/continuous-delivery.md [hunk 1]
```diff
@@ -48,6 +48,40 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
 
 ## March 2024
 
+### Version 1.28.x
+
+#### Fixed issues
+
+- The account level environment broke the GitOps pipeline. (CDS-92546)
```
Comment (roshnisarangadharan) [inline release-notes/continuous-delivery.md:55]:
```suggestion
- The account level environment broke the GitOps pipeline. (CDS-92546, CDS-92569)
```

**Review by SushrutHarness — APPROVED** [review]
LGTM

**Review by shylajasun — APPROVED** [review]

File: release-notes/continuous-delivery.md [hunk 2]
```diff
@@ -48,6 +48,18 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
 
 ## March 2024
 
+### Version 1.28.10
```
Comment (roshnisarangadharan) [inline release-notes/continuous-delivery.md:51]:
```suggestion
### Version 1.28.11
```

File: release-notes/continuous-delivery.md [hunk 3]
```diff
@@ -48,6 +48,18 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+- Fixed an issue where users were unable to search services by ID in service list. (CDS-89587)
+- Continuous Verification for Google Cloud Operations logged error for the `resourceName` field. This issue is fixed by changing the identifier in the request body from `projectId` to `resourceName` for data collection tasks as mentioned in the Google API [documentation](https://cloud.google.com/logging/docs/reference/v2/rest/v2/entries/list). This item requires Harness Delegate version 24.02.82502. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (CDS-89441)
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:60]:
```suggestion
- Continuous Verification for Google Cloud Operations logged error for the `resourceName` field. This issue is fixed by changing the identifier in the request body from `projectId` to `resourceName` for data collection tasks as mentioned in the Google API [documentation](https://cloud.google.com/logging/docs/reference/v2/rest/v2/entries/list). This item requires Harness Delegate version 24.03.82502. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (CDS-89441)  
```

**Review by brian-f-harness — COMMENTED** [review]
Updated the delegate version.

The 1.28.11 was deployed today. https://harness.slack.com/archives/C04R4UDE1FT/p1710307241653819?thread_ts=1710211576.561939&cid=C04R4UDE1FT

https://monitoring.harness.io/d/ZJD3rKZIz/microservice-version-dashboard?orgId=1&refresh=5m&from=now-5m&to=now

**Pull request merged**